sdk/ml/azure-ai-ml/tests/test_configs/pipeline_jobs/serverless_compute/serverless_pipeline_default_compute_valid/pipeline_component.yml

```diff
@@ -0,0 +1,24 @@
+$schema: https://azuremlschemas.azureedge.net/development/pipelineComponent.schema.json
+type: pipeline
+
+name: simple_pipeline_component
+
+jobs:
+  vanilla_node:
+    type: command
+    component: ../components/component.yml
+  node_with_resources:
+    type: command
+    component: ../components/component_with_resources.yml
```

Comment:
This resources is actually in component.
Shall we have a inline command node case?

Reply:
Updated one version, still hold one separate and common command component YAML to reuse across test cases, and overwrite resources in pipeline/pipeline component YAML.